middleware/parseRecipe.js

```diff
@@ -0,0 +1,68 @@
+const formatRecipe = require('./formatRecipe');
+
+const parseRecipe = async url => {
+    const html = await fetch(url)
```

Comment:
just confirming - there shouldn't be CORS errors from these sites in all likelihood